ui/storybook/README.md

```diff
@@ -0,0 +1,122 @@
+# Storybook
+
+This package is used to generate documentation for Perses UI components using [Storybook](https://storybook.js.org/).
+
+> This project currently uses the [v7 beta](https://storybook.js.org/docs/7.0/react/get-started/introduction) of Storybook. Be sure to reference the associated [v7 beta documentation](https://storybook.js.org/docs/7.0/react/get-started/introduction).
```

Comment:
The v7 build is supposed to land soonish, it fixes a handful of issues I noticed when setting up v6.5 (current stable release) in https://github.com/perses/perses/pull/973, and it has better TS support. Considering all of this, I think it's worth us using the beta to start, so we don't have to deal with migrating a ton of stories to the new format. Let me know if you disagree with this.

See the migration doc if you're curious about all the changes:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-65x-to-700

Reply:
I think we should move forward with it. I support using v7